k8s/backend-deployment.yml

```
@@ -0,0 +1,35 @@
+    matchLabels:
+      app: backend
+  template:
+    metadata:
+      name: backend-pod
+      namespace: chat-app
+      labels: 
+        app: backend
+    spec:
+      containers:
+      - name: chatapp-backend
+        image:  swarnendukar123/chatapp-backend:latest
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)